exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,8 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+let sPercentage =(numberOfStudents *100) / 23;
+console.log("Percentage students : " + Math.round(sPercentage) + "%");
+let mPercentage = (numberOfMentors * 100) / 23;
+console.log("Percentage mentors : " + Math.round (mPercentage) + "%");
```

Comment:
best way to put them is in template string - console.log(`Percentage mentors : ${Math.round (mPercentage)} %`)